.devcontainer/rapids.Dockerfile

```diff
@@ -31,6 +30,8 @@ ENV DEFAULT_VIRTUAL_ENV=rapids
 
 ENV RAPIDS_LIBUCX_PREFER_SYSTEM_LIBRARY=true
 
+ENV SCCACHE_IDLE_TIMEOUT=900
+
```

Comment:
I don't think we need this one

Reply:
Why not? Don't we need this set in both containers? Environment variables set for one won't automatically propagate to the other unless the second one uses the first one as its base via `FROM`, right?

Note that I originally tried setting this for both outputs of this Dockerfile by putting this command above the first `FROM` but [that didn't work](https://github.com/rapidsai/devcontainers/actions/runs/12378502809/job/34550538935).

Reply:
There's only one [final build target](https://github.com/rapidsai/devcontainers/blob/71649b53d20bdefd8caec7f431ef4732b3cb6c31/.devcontainer/rapids.Dockerfile#L37-L56), so we can set common envvars there. The build target switches which target stage it extends based on the `PYTHON_PACKAGE_MANAGER` build arg.

Your [initial commit](https://github.com/rapidsai/devcontainers/pull/432/commits/40e5a1cc8b6ce739eeb569532b7bbea88bc2acc4) only set the envvar in the conda build stage, so that's why it only showed up there. We can instead define it once at the bottom, with all the other common sccache envvars.

Reply:
Ohhh nice I missed that there was a final stage there dynamically based off one of the other two. OK that makes a lot of sense then.